system-tests/__snapshots__/results_spec.ts.js

```diff
@@ -31,6 +31,7 @@ exports['module api and after:run results'] = `
     "experimentalSourceRewriting": false,
     "experimentalSingleTabRunMode": false,
     "experimentalStudio": false,
+    "experimentalPromptCommand": false,
```

Comment:
Was requested review as Code Owner due to the results API change here. This change looks good. 👍🏻